automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPage.scala

```diff
@@ -89,38 +94,39 @@ class NotebookPage(override val url: String)(override implicit val authToken: Au
 
   lazy val cells: Query = cssSelector(".CodeMirror")
 
-  def lastCell: WebElement = {
-    webDriver.findElements(cells.by).asScala.toList.last
+  def lastCell: (Int, WebElement) = {
+    val asList = webDriver.findElements(cells.by).asScala.toList
+    (asList.length, asList.last)
```

Comment:
This is probably fine for our tests -- but just noting that the number displayed in `In [n]` is the cell _execution_ order, not necessarily the order they are displayed on the page. But since our tests execute cells one by one, this should be valid I think.

Reply:
Yeah.  I'll add a not with this assumption.